src/gcp_scanner/crawler/cloud_resource_manager_project_info_crawler.py

```diff
@@ -0,0 +1,51 @@
+#  Copyright 2023 Google LLC
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import logging
+import sys
+from typing import Dict, Any
+
+from googleapiclient import discovery
+
+from gcp_scanner.crawler.interface_crawler import ICrawler
+
+
+class CloudResourceManagerProjectInfoCrawler(ICrawler):
```

Comment:
Is there any specific reason to have CloudResourceManager in the name? I'd just omit it and have ProjectInfoCrawler here and everywhere else

Reply:
I was going with the pattern found in #153, like:

![image](https://github.com/google/gcp_scanner/assets/77539004/6817aa51-8cfa-4202-9ae3-c8756660edbc)
![image](https://github.com/google/gcp_scanner/assets/77539004/fa5ed90c-3299-4e11-ab92-7591ce6a98d5)

Both of these have `compute` or `dns` in the prefix of their Class names. Maybe we can also go with `CRM` or put all these in a separate sub-folder to group them without explicitly mentioning it.

Reply:
I see. It is ok, let's keep it. CRM will be confusing